aperture.go

```diff
@@ -333,8 +333,19 @@ func (a *Aperture) Start(errChan chan error) error {
 					"session: %w", err)
 			}
 
+			// An LNC session will automatically try to reconnect to
+			// the node whenever an error occurs. Instead of
+			// shutting the proxy down we will just log the error.
+			lncErrChan := make(chan error)
+			go func() {
+				for {
```

Comment:
This will never quit, causing the goroutine to run forever.
Could just do a range over the channel itself, see https://gobyexample.com/range-over-channels